model/godleyTableWindow.h

```diff
@@ -42,7 +42,7 @@ namespace minsky
   {
     CLASSDESC_ACCESS(ButtonWidget);
   protected:
-    GodleyIcon& godleyIcon;
+    GodleyIcon& godleyIcon;  //reference
```

Comment:
Comment is redundant. Comments should not tell you what the code does, but rather the intent or design. It answer why questions, not how questions.

Reply:
Oops, I forgot that there. I've removed it, thanks for catching it.